coordinator/src/test/scala/filodb.coordinator/queryplanner/ShardKeyRegexPlannerSpec.scala

```diff
@@ -603,7 +603,9 @@ class ShardKeyRegexPlannerSpec extends AnyFunSpec with Matchers with ScalaFuture
       case _ => fail("Expected AggregateMapReduce for the sum operation")
     }
     mpExec.children match {
-      case plan1::plan2::Nil =>
+      case children: Seq[ExecPlan] if children.size == 2 =>
+        val plan1 = children(0)
+        val plan2 = children(1)
```

Comment:
These did not want to work for me. I think it's because `mpExec.children` is now a `Vector` under-the-hood (unclear why that change came about).